views/edit-student.ejs

```diff
@@ -1,90 +1,163 @@
+				<label for="phone_number">Phone Number</label>
+				<input oninput="formatPhoneNumber(this)" id="phoneNumber" type="tel" class="form-control" name="phone_number" pattern ="(?:\(\d{3}\)|\d{3})[- ]?\d{3}[- ]?\d{4}" placeholder="###-###-####" value="<% if (student.phone_number) { %>(<%=student.phone_number.substring(0, 3)%>) <%=student.phone_number.substring(3, 6)%>-<%=student.phone_number.substring(6)%><% } %>" maxlength="14" required <% if (view) { %> readonly <% } %>>
 			</div>
-			<div class="form-group col-md-6">
+			<div class="form-group col-md-2">
+				<label>Country</label>
+				<input name="countryCodeGuardian" class="form-control" list="countryCodes" id="countryCode-guardianPhone" <% if (!add) {%> value="<%=student.countryCodeGuardian%>" <% } else { %> value="+1" <% } %> required <% if (view) { %> readonly <% } %> />
+			</div>
+			<div class="form-group col-md-4">
 				<label for="guardianPhone">Guardian Phone Number</label>
-				<input type="tel" class="form-control" name="guardianPhone" pattern ="[0-9]{3}-[0-9]{3}-[0-9]{4}" placeholder="###-###-####" value="<%=student.guardianPhone%>" required <% if (view) { %> readonly <% } %>>
+				<input oninput="formatPhoneNumber(this)" id="guardianPhoneNumber" type="tel" class="form-control" name="guardianPhone" pattern ="(?:\(\d{3}\)|\d{3})[- ]?\d{3}[- ]?\d{4}" placeholder="###-###-####" value="<% if (student.guardianPhone) { %>(<%=student.guardianPhone.substring(0, 3)%>) <%=student.guardianPhone.substring(3, 6)%>-<%=student.guardianPhone.substring(6)%><% } %>" maxlength="14" required <% if (view) { %> readonly <% } %>>
 			</div>
 			<div class="form-group col-md-6">
 				<label for="notes">Notes</label>
-				<input type="text" class="form-control" name="notes" value="<%=student.notes%>" required <% if (view) { %> readonly <% } %>>
+				<input type="text" class="form-control" name="notes" maxlength = "50" value="<%=student.notes%>" <% if (view) { %> readonly <% } %>>
+			</div>
+			<div class="form-group col-md-6">
+				<label for="dateOfBirth">Date Of Birth</label>
+				<input type="date" class="form-control" name="dateOfBirth" value="<%=student.dateOfBirthFormatted%>" <% if (view) { %> readonly <% } %>>
 			</div>
 			<div class="form-group col-md-6">
 				<label for="interestsAndHobies">Hobbies/Interests.</label>
-				<input type="text" class="form-control" name="interestsAndHobies" value="<%=student.interestsAndHobies%>" <% if (view) { %> readonly <% } %>>
+				<input type="text" class="form-control" name="interestsAndHobies" maxlength = "50" value="<%=student.interestsAndHobies%>" <% if (view) { %> readonly <% } %>>
+			</div>
+			<div class="form-group col-md-6">
+				<label for="grade">Grade</label>
+				<select name="grade" class="form-control" required <% if (view) { %> readonly <% } %>>
+					<% let grades = ["6th", "7th", "8th", "9th", "10th", "11th", "12th", "College Freshman", "College Sophmore", "College Junior", "College Senior", "Out of School"]; %>
```

Comment:
this is where it would be nice to have the grade level list sent up from the back-end